src/openrct2/scripting/ScEntity.hpp

```diff
@@ -176,6 +177,418 @@ namespace OpenRCT2::Scripting
         }
     };
 
+    static const DukEnumMap<VEHICLE_STATUS> VehicleStatusMap({
+        { "moving_to_end_of_station", VEHICLE_STATUS_MOVING_TO_END_OF_STATION },
+        { "waiting_for_passengers", VEHICLE_STATUS_WAITING_FOR_PASSENGERS },
+        { "waiting_to_depart", VEHICLE_STATUS_WAITING_TO_DEPART },
+        { "departing", VEHICLE_STATUS_DEPARTING },
+        { "travelling", VEHICLE_STATUS_TRAVELLING },
+        { "arriving", VEHICLE_STATUS_ARRIVING },
+        { "unloading_passengers", VEHICLE_STATUS_UNLOADING_PASSENGERS },
+        { "travelling_boat", VEHICLE_STATUS_TRAVELLING_BOAT },
+        { "crashing", VEHICLE_STATUS_CRASHING },
+        { "crashed", VEHICLE_STATUS_CRASHED },
+        { "travelling_dodgems", VEHICLE_STATUS_TRAVELLING_DODGEMS },
+        { "swinging", VEHICLE_STATUS_SWINGING },
+        { "rotating", VEHICLE_STATUS_ROTATING },
+        { "ferris_wheel_rotating", VEHICLE_STATUS_FERRIS_WHEEL_ROTATING },
+        { "simulator_operating", VEHICLE_STATUS_SIMULATOR_OPERATING },
+        { "showing_film", VEHICLE_STATUS_SHOWING_FILM },
+        { "space_rings_operating", VEHICLE_STATUS_SPACE_RINGS_OPERATING },
+        { "top_spin_operating", VEHICLE_STATUS_TOP_SPIN_OPERATING },
+        { "haunted_house_operating", VEHICLE_STATUS_HAUNTED_HOUSE_OPERATING },
+        { "doing_circus_show", VEHICLE_STATUS_DOING_CIRCUS_SHOW },
+        { "crooked_house_operating", VEHICLE_STATUS_CROOKED_HOUSE_OPERATING },
+        { "waiting_for_cable_lift", VEHICLE_STATUS_WAITING_FOR_CABLE_LIFT },
+        { "travelling_cable_lift", VEHICLE_STATUS_TRAVELLING_CABLE_LIFT },
+        { "stopping", VEHICLE_STATUS_STOPPING },
+        { "waiting_for_passengers_17", VEHICLE_STATUS_WAITING_FOR_PASSENGERS_17 },
+        { "waiting_to_start", VEHICLE_STATUS_WAITING_TO_START },
+        { "starting", VEHICLE_STATUS_STARTING },
+        { "operating_1a", VEHICLE_STATUS_OPERATING_1A },
+        { "stopping_1b", VEHICLE_STATUS_STOPPING_1B },
+        { "unloading_passengers_1c", VEHICLE_STATUS_UNLOADING_PASSENGERS_1C },
+        { "stopped_by_block_brake", VEHICLE_STATUS_STOPPED_BY_BLOCK_BRAKES },
```

Comment:
At some point we should strive for a `toString()` of `fromString()` method, because if a value is added to the enum currently, we won't be able and multiple places will be left unhandled. If it's a `switch()...case` it would error out.